frontend/src/AppInbusImport.vue

```diff
@@ -0,0 +1,327 @@
+<script setup lang="ts">
+import { ref, watch, onMounted } from 'vue';
+
+import { csrfToken } from './api.js';
+
+class WeekActivity {
+  /**
+    Výskyt aktivity v konkrétním týdnu výuky.
+    */
+
+  weekActivityId: number; // ID výskytu aktivity v konkrétním týdnu výuky
+  weekNumber: number; // číslo týdne v semestru
+  date: string; // datum výskytu aktivity (yyyy-MM-dd)
+}
+
+class ConcreteActivity {
```

Comment:
Could you please put these type definitions into a separate file, e.g. `inbus.ts`?

Reply:
Done!